### ISSUE TITLE
Draw is unable to open an SVG diagram properly

### ISSUE THREAD

**Issue by raindropsfromsky** [jgraph/drawio-desktop#407]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**

Draw is unable to open the following SVG diagram.
https://www.yworks.com/assets/images/ygucd/am.b0334968.svg

**To Reproduce**
1. Save the SVG diagram on local disk.
2. Open it in a browser to check if it is ok. Zoom in to check that it does not blur.
3. Import the same diagram in Draw (using the **File>Import** menu)
4. The diagram appears as a blurred thumbnail.
![image](https://user-images.githubusercontent.com/9047168/96365644-b0c1fa80-115f-11eb-982b-53514983aed8.png)

If we zoom in, the blurriness increases.

**Expected behavior**
An SVG diagram is a vector diagram, and so it should remain crystal clear at any zoom level, and not blur like a raster diagram.

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version [e.g. 13.7.9]

**Desktop (please complete the following information):**
 - OS: Windows 10, Hiome edition (64 bit)
 
**Additional context**
I created a flowchart in Draw, and exported it as SVG. Then I imported the SVG file in Draw.
It shows the diagram properly.
So logically it should display any SVG diagram properly.

**Comment by mararad** [comment]
My suggestion would be to drag&drop SVG into the editor and check if it is blurred. File > Import results with blur image.

Regards,

**Comment by raindropsfromsky** [comment]
Drag-n-drop works. The image is not blurred.
But that's only a workaround: The import does not work, as reported.

**Comment by hohwille** [comment]
FYI: This is still all very strange: 
* Import of SVG seems to import as bitmap
* drag&drop SVG into drawio seems to link the SVG as image into drawio diagram

What I really need is to edit the SVG with drawio so I can ungroup shapes, edit texts, etc.
However this is not supported and I have to redraw the entire SVG from scratch with drawio.
Also copy&pasting shapes from Inkscape, BoxySvg, Visio, Dia, or any other vector drawing program into drawio is not possible. That is sad...

**Comment by mattwilmott** [comment]
If drawio were able to open/import an SVG exported from Inkscape, Visio or even PowerPoint, the tool would be an order of magnitude more useful. I am often presented with flow/governance/network/systems diagrams as SVG images exported from these toolsets and would love to work with them rather than redrawing.

I can see others have similar requirements, https://arlanblogs.alvarnet.com/converting-azure-architecture-powerpoint-slides-to-diagrams-net-draw-io-platform/

**Comment by davidjgraph** [comment]
To be clear, draw.io is not an SVG editing tool. If that's the intention of this request, afraid that's out of scope of this project. But, the thread is about SVGs being imported as a bitmap.

In terms of that original problem, we need an example SVG and the environment details that recreate the problem.

**Comment by EricPotratz** [comment]
I'm having the same problem.

All drawing programs should be able to import/export/edit SVG files since they are just XML files. I've tried uploading my SVG file to draw.io (Diagrams.net) and it only uploads a static image. Is there anyway to upload an SVG so it can be edited like any other XML file?

I had to change the extension to .jpg to upload the file. It will need to be changed back to .svg:
![Palm Fruit research timeline diagram](https://user-images.githubusercontent.com/32938127/135688016-7ebd30e4-a46f-4ccd-a7a3-415467794f39.jpg)
